.github/workflows/integration.yml

```diff
@@ -45,7 +45,7 @@ jobs:
       - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11
         with:
           repository: kubernetes-sigs/gateway-api
-          ref: 4f86f0bd65173b04dadb558f63fbbd53330736d2 # 0.5.0-rc1
+          ref: a0684982eddeb0360e215e0de322c3210ac49bb9 # v1.0.0
```

Comment:
I don't think we want to take the v1 update this change. That should be made distinctly.

Note that the v1 change necessarily breaks API compat for this crate, while the gRPC addition should not need to.

Reply:
🤔 I'm not sure where that ref change from `0.5.` to `1.0.0` came from. Something may be goofed up somewhere, I don't _think_ I would have fiddled with anything in the `.github/` directory, and there's also a whole `duration.rs` file I have no recollection of.

Let me investigate before we push forward here.

Reply:
@olix0r Found it - looks like the change actually came from [this commit](https://github.com/linkerd/k8s-gateway-api-rs/pull/94/commits/ea8c0447edfdebfe5da6bc3e76b270ee9c0f999c) from a couple of days ago.

If it's a breaking change, what do you want to do here?

Reply:
I suppose you're missing 9253b82147bb5d043626fd24964aced18964411e in your history? It looks like you should rebase or merge main.

Reply:
@olix0r I _genuinely_ have no idea why's been going on with my branch histories. I _suspect_ it's silliness with having to go back to re-issue `--signoff` (which is just _infuriating_ seeing as I've got `signOff = true` set in my global git config, but that's an issue for a different day).

I've fixed up the branch and its history, so it _should_ be good now. Would you mind taking a look again?